lib/cartopy/crs.py

```diff
@@ -156,6 +156,41 @@ def _determine_longitude_bounds(self, central_longitude):
             minlon += epsilon
         return minlon, maxlon
 
+    def _repr_svg_(self):
+        """
+        Make a visual representation of the projection and return it as an
+        html element.
+
+        """
+        # Imports.
+        try:
+            # As matplotlib is not a core cartopy dependency, don't error
+            # if it's not available, but fall back to the default repr.
+            import matplotlib.pyplot as plt
+        except ImportError:
+            pass
+        else:
+            if six.PY2:
+                from StringIO import StringIO
+            else:
+                from io import StringIO
+            # Produce a visual repr of the Projection instance.
+            fig, ax = plt.subplots(figsize=(5, 3),
+                                   subplot_kw={'projection': self})
+            ax.set_global()
+            ax.coastlines('110m')
```

Comment:
You could try out the 'auto' scale... might produce a more desirable result for limited area maps.